extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ClassConfigPropertiesUtil.java

```diff
@@ -316,6 +322,15 @@ private static ResultHandle populateConfigObject(ClassLoader classLoader, ClassI
         return configObject;
     }
 
+    private static String getPropertyName(FieldInfo field, ConfigProperties.NamingStrategy namingStrategy) {
+        if (namingStrategy == ConfigProperties.NamingStrategy.DEFAULT) {
+            return field.name();
+        } else if (namingStrategy == ConfigProperties.NamingStrategy.KEBAB_CASE) {
+            return StringUtil.hyphenate(field.name());
+        }
```

Comment:
Maybe move this behavior to a method in the NamingStrategy enum? eg. `String getPropertyName(String name)`

Reply:
That makes sense, I'll update it soon

Reply:
That makes sense, I'll update it soon

Reply:
Done